style.css

```diff
@@ -0,0 +1,136 @@
+html {
+  box-sizing: border-box;
+  font-size: 16px;
+  font-family: 'Inter', sans-serif;
+  scroll-behavior: smooth;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+body {
+  background-color: #fff;
+  margin: 0 auto;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 27px;
+  color: #acb7c3;
+}
+
+.hamburger {
+  display: block;
+  cursor: pointer;
+}
+
+.slide {
+  display: block;
+  height: 3px;
+  width: 25px;
+  margin: 5px auto;
+  background-color: #acb7c3;
+}
+
+.content {
+  background-image: url(./images/mobile/header-illsutration-mobile.svg);
+  background-repeat: no-repeat;
+  background-position-x: 100%;
+}
+
+.presentation {
+  margin: 0 27px;
+  text-align: center;
+  padding-top: 65px;
+}
+
+.presentation-text {
+  margin: 15px 0;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  text-align: center;
+  line-height: 28px;
+}
+
+.presentation > h2 {
+  padding: 0 3rem;
+}
+
+.title {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  margin-top: 0;
+  margin-bottom: 0;
+  line-height: 54px;
+}
+
+.dtitle {
+  margin-top: 0;
+  color: #36b37e;
+}
+
+.social-Icons {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  padding: 0;
+}
+
+.circle-container {
+  display: flex;
+  justify-content: center;
+  padding: 70px 0;
+}
+
+.circle-fade {
+  width: 40px;
+  height: 40px;
+  position: relative;
+}
+
+.circle-fade i {
+  font-size: 24px;
+  color: #000;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.circle-fade::before {
+  content: "";
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: linear-gradient(transparent, black);
+  opacity: 0.2;
+  animation: fade 2s infinite;
+}
+
+/* Defines the animation fro the fade effect */
+@keyframes fade {
+  0% {
+    opacity: 0.5;
+  }
+
+  50% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 0.5;
+  }
+}
```

Comment:
- According to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Minimum%20desktop%20screen%20size%20should%20be%20768px), your page is supposed to have a Minimum desktop screen size should be 768px. Please Kindly fix that and add responsiveness to your website.